theme/parts/tabsbar.css

```diff
@@ -316,3 +316,70 @@ tab[selected]:-moz-window-inactive {
 		border-bottom-color: var(--gnome-tabbar-tab-active-border-bottom-color-contrast) !important;
 	}
 }
+
+/* OPTIONAL: Use HdyTabBar styling */
+@supports -moz-bool-pref("gnomeTheme.newTabs") {
+
+	/* Change the tab styling */
+
+	/* Active tab */
+	.tab-background[selected=true] {
+		background-color: var(--gnome-tabbar-tab-active-background-new) !important;
+		border-right: 1px !important;
+		border-bottom: 1px solid var(--gnome-tabbar-tab-border-color-new) !important;
+		transition: background-color 2s ease-out 100ms;
+	}
+	/* Hover */
+	:root:not(:-moz-window-inactive) .tabbrowser-tab:hover > .tab-stack > .tab-background[selected=true] {
+		background-color: var(--gnome-tabbar-tab-active-hover-background-new) !important;
+	}
+	/* Inactive tab */
+	#TabsToolbar {
+		background-color: var(--gnome-tabbar-tab-background-new) !important;
+	}
+	/* Hover */
+	:root:not(:-moz-window-inactive) .tabbrowser-tab:hover > .tab-stack > .tab-background:not([selected=true]) {
+		background-color: var(--gnome-tabbar-tab-hover-background-new) !important;
+		border-bottom: 1px solid var(--gnome-tabbar-tab-border-color-new) !important;
+	}
+
+
+	/* Adjust Borders */
+
+	tab, tab[selected] {
+		border-left: 0px !important;	/* this is a bodge */
+		border-right: 1px solid var(--gnome-tabbar-tab-border-color-new) !important;	
+	}
+
+	/* Adjust Margins */
+
+	.tabbrowser-tab[first-tab]:not([pinned=true]), .tabbrowser-tab[last-tab]:not([pinned=true]), .tabbrowser-tab:not([pinned=true]) {
+		margin-left: 0px !important;
+		margin-right: 0px !important;
+	}
+
+
+	/* Close Button Styling */
+
+	.tab-close-button {
+		border-radius: 99px;
+		opacity: 1;
+	}
+	:root:not(:-moz-window-inactive) .tab-close-button:hover {	
+		background-image: none;
+		background-size: 24px;
+		background-color: var(--gnome-tabbar-close-hover);
+		box-shadow: 0px 0px transparent; 
+	    border: 0px solid transparent;
```

Comment:
This should be applied to `.tab-close-button` it self or will do a jump on hover.